Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -691,357 +716,400 @@ def create_ioc_request(self, name, source, ioc_type, method, validUntil, value,
         query_params = assign_params(
             skip=0,
             limit=30,
             sortOrder="desc",
         )
-        response = self._http_request(method='GET', url_suffix=endpoint_url, params=query_params,
-                                      retries=3, backoff_factor=5, status_list_to_retry=[200, 202])
+        response = self._http_request(
+            method="GET",
+            url_suffix=endpoint_url,
+            params=query_params,
+            retries=3,
+            backoff_factor=5,
+            status_list_to_retry=[200, 202],
+        )
         urls_found = []
         data = []
         if response["data"] is not None:
             data = response["data"]
         for i in data:
-            if i['data'].get('downloadUrl') is not None:
-                urls_found.append(i['data'].get('downloadUrl'))
+            if i["data"].get("downloadUrl") is not None:
+                urls_found.append(i["data"].get("downloadUrl"))
         for item in urls_found:
             if item[:8] == "/agents/":
                 return item
         return "-1"
 
     def get_alerts_request(self, query_params):
-        endpoint_url = 'cloud-detection/alerts'
+        endpoint_url = "cloud-detection/alerts"
 
-        response = self._http_request(method='GET', url_suffix=endpoint_url, params=query_params)
-        alerts = response.get('data', {})
-        pagination = response.get('pagination')
+        response = self._http_request(method="GET", url_suffix=endpoint_url, params=query_params)
+        alerts = response.get("data", {})
+        pagination = response.get("pagination")
         return alerts, pagination
 
     def download_threat_file_request(self, endpoint_url):
-        return self._http_request(method='GET', url_suffix=endpoint_url, resp_type='content')
+        return self._http_request(method="GET", url_suffix=endpoint_url, resp_type="content")
 
     def get_installed_applications_request(self, query_params):
-        endpoint_url = 'agents/applications'
-        response = self._http_request(method='GET', url_suffix=endpoint_url, params=query_params)
-        return response.get('data', [])
+        endpoint_url = "agents/applications"
+        response = self._http_request(method="GET", url_suffix=endpoint_url, params=query_params)
+        return response.get("data", [])
 
     def initiate_endpoint_scan_request(self, agent_ids):
-        endpoint_url = 'agents/actions/initiate-scan'
+        endpoint_url = "agents/actions/initiate-scan"
+        payload = {"filter": {"ids": agent_ids}, "data": {}}
+        response = self._http_request(method="POST", url_suffix=endpoint_url, json_data=payload)
+        return response.get("data", {})
+
+    def get_s1_threats_information(self, threat_ids):
+        response = self._http_request(method="GET", url_suffix=f"threats?ids={threat_ids}")
+        return response.get("data", [])
+
+    def get_s1_threats_request(self, last_update):
```

Comment:
No need to create a new request, there already such a request that you should use (`get_threats_request`).